_posts/2019-02-21-february-meetup.md

```diff
@@ -1,3 +1,19 @@
+---
+title: February 21st
+type: meetup
+location: Karnov Group Denmark A/S, Sankt Petri Passage 5 · København
```

Comment:
Great 👍 , you just need to remove the dot that meetup has (which makes no sense why they would use, zero ux)